jerry-core/ecma/builtin-objects/typedarray/ecma-builtin-typedarray-prototype.c

```diff
@@ -1891,7 +1891,7 @@ ecma_builtin_typedarray_prototype_copy_within (ecma_value_t this_arg, /**< this
   int32_t offset = (int32_t) (length - target);
   int32_t count = JERRY_MIN (distance, offset);
 
-  if (target >= length || start >= length || end == 0)
+  if (target >= length || start >= length || end == 0 || start >= end)
```

Comment:
Shouldn't the `start >= length` condition just be replaced with `start >= end`? It seems to me that `end` is always <= `length`.

Reply:
Thats a good point, i will fix it!